packages/frontend/web-ui/src/app/store/features/authSlice.spec.js

```diff
@@ -129,7 +129,7 @@ describe('authSlice', () => {
     beforeAll(() => {
```

Comment:
This file should be migrated as well

Reply:
Done! But... I'm still working with the `createAuthByToken` and `createAuthByCredentials` tests to convert them into TS.

Reply:
Done in both tests!